docker/Dockerfile.distroless

```diff
@@ -0,0 +1,20 @@
+# prepare a distroless source context to copy files from
+FROM gcr.io/distroless/static-debian10@sha256:4433370ec2b3b97b338674b4de5ffaef8ce5a38d1c9c0cb82403304b8718cde9 as distroless_source
+
+# prepare a base dev to modify file contents
+FROM docker.io/istio/base:1.7-dev.1 as ubuntu_source
```

Comment:
Providing a base version will help to update it quickly when we need to change it to the latest version and it will be aligned with other Dockerfile. Not sure if `ARG` is required here?
 
```Dockerfile
ARG BASE_VERSION=1.7-dev.1
FROM docker.io/istio/base:${BASE_VERSION} as ubuntu_source
```

Reply:
I addressed this by using "from ubuntu:bionic". There is no strong correlation between the need as ubuntu for a source image and the resulting image. I pull the data from the distroless image. Ubuntu is only used to transform the `passwd` file into something that is useful.

Cheers,
-steve